src/model.py

```diff
@@ -0,0 +1,85 @@
+
+import torch
+import torch.nn as nn
+from catalyst.dl import registry
+
+
+@registry.Model
```

Comment:
I think we do not need register model here since we already do it in `__init__.py`